docs/source/examples/checks/methodology/single_feature_contribution_train_test.ipynb

```diff
@@ -5,123 +5,83 @@
    "id": "21c5d5bc",
```

Comment:
<p>The first sentence should be more "motivation of this check" / what it may indicate, and only then the more technical stuff....</p><p> </p><p>+1 to Bressler - missing here content - what is PPS, how does it work, intuition, etc.</p><p>The explanation that is built-in the check itself is more elaborate and clear then this one, it should be the other way around, people that come to this notebook want to understand more.</p><p> </p><p>and also - link (opened in a new tab) to an external resource, explaining about pps, for those that want to read even more...?</p><p> </p><p>*Also - relate to "correlation" - why don't we just use the standard and well-known correlation metrics for the purpose of "Single Feature Contribution"</p><p> </p><p>--</p><p>(also some typos but better to comment about that after the rephrasing)</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>There is a link to the ppscore package, I don't think it's the place to explain about concept that is not ours</p>

Reply:
<p>PPS is a concept that is unfamiliar to most DS... so we really should elaborate about it and not "send them searching"... this is one of the main purposes of these notebooks. of course, we should give the due credit to the creator, but that doesn't mean we shouldn't explain - what is it, how does it work, why we're using it... especially since these are questions that are frequently raised regarding this check (e.g. "what is this, how does this work, why aren't you using correlation instead...)</p><p> </p><p>link-wise, you can also consider this link -</p><p>https://towardsdatascience.com/rip-correlation-introducing-the-predictive-power-score-3d90808b9598 (that's currently in the check itself). generally speaking we want to keep users in our docs, and give them all of the information needed to keep them satisfied.</p>